ibis/backends/bigquery/__init__.py

```diff
@@ -670,16 +670,32 @@ def _get_schema_using_query(self, query: str) -> sch.Schema:
         return BigQuerySchema.to_ibis(job.schema)
 
     def raw_sql(
-        self, query: str, params=None, job_id: str | None = None
+        self,
+        query: str,
+        params: dict | None = None,
+        job_id: str | None = None,
+        query_job_props: dict | None = None,
```

Comment:
Let's call it `query_job_config` to match the Google API naming convention.

Reply:
I updated to a different approach which is better. My rationale for using the `props` naming is no longer relevant, so I also updated to `query_job_config`.